scripts/deploy/nlp/query.py

```diff
@@ -38,6 +38,7 @@ def get_args(argv):
     parser.add_argument("-tpp", "--top_p", default=0.0, type=float, help="top_p")
     parser.add_argument("-t", "--temperature", default=1.0, type=float, help="temperature")
     parser.add_argument("-ti", "--task_id", type=str, help="Task id for the prompt embedding tables")
+    parser.add_argument("-lp", "--log_probs", default=False, action='store_true', help="Returns log_probs")
```

Comment:
@oyilmaz-nvidia maybe unrelated to this PR, but curious what's the difference between this script and [nemo/deploy/nlp/query_llm.py](https://github.com/NVIDIA/NeMo/blob/e9d73742dccfc3f6b77946bd0440d4787c73f7ce/nemo/deploy/nlp/query_llm.py) ?

Reply:
Script is for those who would like to use the script and it doesn't require nemo installation. And the class is there for nemo users who want to use python classes.